src/models/expenseModel.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-console */
+/* eslint-disable function-paren-newline */
+/* eslint-disable no-shadow */
+'use strict';
+
+const expenses = [];
```

Comment:
IT would be better for performing CRUD operations if _expenses_ were some sort of Hashmap, like an Object or Map instance